app/Library/Notifications/RecipientFinder.php

```diff
@@ -59,6 +61,21 @@ public function publishedNewTrack(Track $track) {
         switch ($this->notificationDriver) {
             case PonyfmDriver::class:
                 return $track->user->followers;
+            case NativeDriver::class:
+                $followerIds = [];
+                $subIds = [];
+                $rawSubIds = Subscription::select('id')->get();
+
+                foreach ($track->user->followers as $follower) {
+                    array_push($followerIds, $follower->id);
+                }
+
+                foreach ($rawSubIds as $sub) {
+                    array_push($subIds, $sub->id);
```

Comment:
Use `->lists('name_of_column')` on a query to get an array of column values right out of the query. ctrl+F on this page for "retrieving a list of column values": https://laravel.com/docs/5.1/queries